### PR TITLE
Filling documentation index page

### PR DESCRIPTION
**Pull request by vfricou**
- Adding some words about project
- Adding some words about us

### CODE REVIEW

**Comment by vfricou** [comment]
I think will done an orthograph check. I've write this quite speedly and I've probably make some fails.

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
 # SillyWhale docker images
 
-Our goal is to create lights and secure docker's images for somes opensources applications.
\ No newline at end of file
+Welcome on our documentation website.
+
+Here you could be found documentation of any [Docker](https://docker.io) image we provide on [store](https://store.docker.com/) and throught our [GitHub repository](https://github.com/SillyWhale).
```
Comment (tsarti) [inline docs/index.md]: Here **you can find documentation about** any [Docker](https://docker.io) image we provide on [store](https://store.docker.com/) and **through** our [GitHub repository](https://github.com/SillyWhale).

File: docs/index.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+For each image, you'll found a section for it. And if is necessary, we'll create subsection to explain container content.
```
Comment (tsarti) [inline docs/index.md]: For each image, **you'll find** a section **about** it. And if **it's** necessary, we'll create **subsections** to explain **container's** content.

File: docs/index.md (hunk 1 above)
Comment (tsarti) [inline docs/index.md]: Here **you can find documentation about** any Docker image we provide on store and **through** our GitHub repository.

File: docs/index.md (hunk 2 above)
Comment (tsarti) [inline docs/index.md]: For each image, you'll **find** a section **about** it. And if **it's** necessary, we'll create **subsections** to explain **container's** content.

File: docs/index.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+
+According with all we says in [our repository](https://github.com/SillyWhale/_management/blob/master/CONTRIBUTING.md), please be indulgent, we aren't many persons to maintain theses images and this is not our primary work. So, thank four our patience and your kindness.
+
+## About us
+
+We're little friends group, we're passionate by IT and all news technologies in this domain.  
+We some recently work on [Docker](https://docker.io) containers, and we have a so bad analysis of existings packaged apps. (Not all fornunatly)
```
Comment (tsarti) [inline docs/index.md]: We recently **worked** on [Docker](https://docker.io) containers, and we have a **real** bad analysis of **existing packaged apps**. (Not all of them fortunately)

File: docs/index.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+
+According to this constatations, we've decided to launch [SillyWhale](https://www.sillywhale.wtf) project to provide our help to persons who aren't be confortable with this tech.
```
Comment (tsarti) [inline docs/index.md]: According to **these** constatations, we've decided to launch [SillyWhale](https://www.sillywhale.wtf) project to provide our help to persons who **aren't confortable** with this tech.

File: docs/index.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+
+Our goals is some easier to understand :
```
Comment (tsarti) [inline docs/index.md]: Our goals **are rather easy** to understand :

File: docs/index.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+
+- Build docker image
+- Pass good moments to work on it
```
Comment (tsarti) [inline docs/index.md]: **Have a good time working on them**

File: docs/index.md [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+## About us
+
+We're little friends group, we're passionate by IT and all news technologies in this domain.  
+We some recently work on [Docker](https://docker.io) containers, and we have a so bad analysis of existings packaged apps. (Not all fornunatly)
+
+According to this constatations, we've decided to launch [SillyWhale](https://www.sillywhale.wtf) project to provide our help to persons who aren't be confortable with this tech.
+
+Our goals is some easier to understand :
+
+- Build docker image
```
Comment (tsarti) [inline docs/index.md]: Build docker **images**

File: docs/index.md [hunk 8]
```diff
@@ -1,3 +1,25 @@
...
+- Pass good moments to work on it
+- Not realy more
```
Comment (tsarti) [inline docs/index.md]: Not **really** more

File: docs/index.md [hunk 9]
```diff
@@ -1,3 +1,25 @@
...
+
+But, we don't want build [Docker](https://docker.io) image and let it in bulk versions on [Docker store](https://store.docker.com/). We have make a choice to full document any image we build, and we've make a second choice, to secure at maximum of us competence these images.
```
Comment (tsarti) [inline docs/index.md]: But, we don't want **to** build [Docker](https://docker.io) **images** and let **them** in bulk versions on [Docker store](https://store.docker.com/). **We've made the choice to fully document** any image we build, and **we've made the second choice**, **to secure them at the maximum of our competences**.

File: docs/index.md [hunk 10]
```diff
@@ -1,3 +1,25 @@
...
+According to this constatations, we've decided to launch [SillyWhale](https://www.sillywhale.wtf) project to provide our help to persons who aren't be confortable with this tech.
+
+Our goals is some easier to understand :
+
+- Build docker image
+- Pass good moments to work on it
+- Not realy more
+
+But, we don't want build [Docker](https://docker.io) image and let it in bulk versions on [Docker store](https://store.docker.com/). We have make a choice to full document any image we build, and we've make a second choice, to secure at maximum of us competence these images.  
+Surely, we not redevelop applications we package and in this condition, we couldn't be garant to application internal security, but we make our maximum for all we can touch around.
```
Comment (tsarti) [inline docs/index.md]: **Clearly, we don't redevelop** applications **but only package them**, and in **this conditions**, we couldn't be garant of **application's internal security**, but **we will make our maximum with**  all we can touch around.

Comment (vfricou) [reply docs/index.md]: Here, you change sens of sentence. I prefer keep this same as origin : " […] applications we package and in this condition, we couldn't be garant to application internal security […] "

Comment (tsarti) [reply docs/index.md]: I don't think I'm changing the meaning of your sentence. Maybe, I didn't understand what you wanted to say and made bad changes.

File: docs/index.md [hunk 11]
```diff
@@ -1,3 +1,25 @@
...
 
-Our goal is to create lights and secure docker's images for somes opensources applications.
\ No newline at end of file
+Welcome on our documentation website.
+
+Here you could be found documentation of any [Docker](https://docker.io) image we provide on [store](https://store.docker.com/) and throught our [GitHub repository](https://github.com/SillyWhale).
+
+For each image, you'll found a section for it. And if is necessary, we'll create subsection to explain container content.
+
+According with all we says in [our repository](https://github.com/SillyWhale/_management/blob/master/CONTRIBUTING.md), please be indulgent, we aren't many persons to maintain theses images and this is not our primary work. So, thank four our patience and your kindness.
```
[the remark this answers is not in the document]
Comment (tsarti) [reply docs/index.md]: @vfricou "According **_to_** all we say" 
You forgot this change

File: docs/index.md (hunk 3 above)
Comment (tsarti) [reply docs/index.md]: @vfricou "all of them **_fortunately_**"
You forgot this change

File: docs/index.md (hunk 9 above)
Comment (tsarti) [reply docs/index.md]: @vfricou "we don't want **_to_** build"
You forgot this change

**Pull request merged**